### PR TITLE
prepare release 2.3.3

### PR DESCRIPTION
**Pull request by antban**
ARUHA-1342, ARUHA-1375, ARUHA-1373

Changelog for release

### CODE REVIEW

**Comment by antban** [comment]
:+1:

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+
```
Comment (v-stepanov) [inline CHANGELOG.md]: excess empty line

Comment (antban) [reply CHANGELOG.md]: @v-stepanov I hate this file

**Comment by antban** [comment]
:+1:

**Comment by antban** [comment]
@v-stepanov fixed: https://github.com/zalando/nakadi/pull/800/commits/1a1c07ee627ce75d9420070c7fa3a508af630c85

**Comment by antban** [comment]
:+1:

**Comment by lmontrieux** [comment]
👍

**Comment by adyach** [comment]
👍

**Pull request merged**